frontend/public/components/_resource.scss

```diff
@@ -118,6 +118,10 @@
     display: inline-flex;
     margin: 0 20px 0 0;
   }
+  &--inline-close {
```

Comment:
I'm confused why we need two `inline` styles. I understand you're adjusting the margin, but what does the normal `inline` style look like and why is the right margin so large that it won't work here? Is it used in a different context?

Reply:
Hi,
Normal `inline` is used in the Operands Link in the `OperandsTable`:
![image](https://user-images.githubusercontent.com/12733153/130289012-19d29719-60a5-4e00-a90f-120e38e7059d.png)

`inline-close` is used when there is an error with deleting an operand and is shown in the error alert:
![image](https://user-images.githubusercontent.com/12733153/130288824-2db44d6e-d993-41a0-a4ab-85e5c078a40a.png)

Using just `inline` on the error alert causes a large right side gap:
![image](https://user-images.githubusercontent.com/12733153/130287873-f581f29c-37d6-4f79-baee-5e079b593ef9.png)

...and I don't know why the list is suddenly showing a bullet :-)  Something change with bootstrap loading?
![image](https://user-images.githubusercontent.com/12733153/130289385-edd32e4c-ac40-4874-afef-54fa85841f2f.png)

Reply:
Hi @spadgett, However, if we decide to use use https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/utils/promise.ts#L21 then we'd just display the error messages and we wouldn't need the resource link anymore.

Reply:
I guess I'm asking why the normal inline style needs the right margin? Is it used other places or only in the table?

Reply:
`co-resource-item--inline` is used a bunch of places, for example [here in /frontend/packages/dev-console/src/components/health-checks/AddHealthChecks.tsx](https://github.com/openshift/console/blob/5277de63ed7b371dd30faed5a52bcd25fcbc9027/frontend/packages/dev-console/src/components/health-checks/AddHealthChecks.tsx#L100).  

I'm wondering if we even need the `--inline` flavor, looks fine not inline:
![image](https://user-images.githubusercontent.com/12733153/130672272-c8de5cee-ed82-4655-8f37-4747f50bdc45.png)

...or even the default `--inline` looks ok:
![image](https://user-images.githubusercontent.com/12733153/130672499-d7c46a51-cd2f-43fc-9446-727f57e4a248.png)

Although, I do like the `--inline-close` the most.

Reply:
@dtaylor113 Personally I think it looks better without the `--inline` from your screenshots.

Reply:
removed `--inline` usage.